doc/en/developer/source/source.rst

```diff
@@ -12,28 +12,128 @@ To clone the repository::
 To list available branches in the repository::
 
   % git branch
-     2.1.x
-     2.2.x
+     2.15.x
+     2.16.x
    * master
 
-To switch to the stable branch::
+To switch to the 2.16.x branch above::
 
-  % git checkout 2.2.x
+  % git checkout 2.16.x
   
 Git
 ---
 
-Those coming from a Subversion or CSV background will find the Git learning curve is a steep one.
+Git is a distributed version control system with a steep learning curve.
 Luckily there is lots of great documentation around. Before continuing developers should take the 
 time to educate themselves about git. The following are good references:
 
-* `The Git Book <http://git-scm.com/book/>`_
-* `A nice introduction <http://www.sbf5.com/~cduan/technical/git/>`_
+* `The Git Book <http://git-scm.com/book/>`__
+* `A nice introduction <http://www.sbf5.com/~cduan/technical/git/>`__
+* `Git Pull Requests <https://help.github.com/en/articles/about-pull-requests>`__
+
+.. _gitconfig:
+
+Git client configuration
+------------------------
+
+To review global settings:
+
+.. code-block:: bash
+
+   $ git config --global --get-regexp core.*
+
+::
+
+   core.autocrlf input
+   core.safecrlf true
+   core.ignorecase true
```

Comment:
If thise setting does not make sense for case sensitive file systems please mark it as a separate setting in a note.

Reply:
I will provide a second example then.

Reply:
I have split this into two examples, and after research/reflection am now asking Mac and Windows users to set this value to `true` (so even though their file system does not care - we are asking the gitclient to care so that files work correctly on Linux).

I also provided an example of how to fix any mistakes using `git mv`.